jaggeryapps/admin/serviceproviders/editsp.jag

```diff
@@ -141,17 +141,16 @@ include("../templates/leftpane.jag");
               <div class="btn-group" id="custom-app-dropdown" style="display: none;">
                   <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#" id="app-type-dropdown">Select App Type<span
                           class="caret"></span></a>
-                  <ul class="dropdown-menu">
-                      <li><a id="agenttype" href="#">Agent</a></li>
-                      <li><a id="proxytype" href="#">Proxy</a></li>
+                  <ul class="dropdown-menu" id="app-type-drop-down">
+                      <li><a id="agenttype" href="#">Standards-Based Federation</a></li>
```

Comment:
Should this be standards-based or standard-based ? Can you please verify with someone from doc team